README.md

```diff
@@ -1,9 +1,17 @@
 <p align="center"><a href="https://modin.readthedocs.io"><img width=77% alt="" src="https://github.com/modin-project/modin/blob/3d6368edf311995ad231ec5342a51cd9e4e3dc20/docs/img/MODIN_ver2_hrz.png?raw=true"></a></p>
 <h2 align="center">Scale your pandas workflows by changing one line of code</h2>
 
+<div align="center">
+
+Dev Community & Support (<1 day response) | Forums | Socials |
+|:---: | :---: | :---: |
+| <a href='https://join.slack.com/t/modin-project/shared_invite/zt-yvk5hr3b-f08p_ulbuRWsAfg9rMY3uA'><img src='https://img.shields.io/static/v1?label=chat&logo=slack&message=Slack&color=brightgreen' alt='Slack' height=28/></a>     | <a href="https://discuss.modin.org"><img alt="" src="https://img.shields.io/badge/discourse-forum-purple.svg?logo=discourse&logoColor=white" height=28></a> [![Stack Overflow](https://img.shields.io/badge/-Stackoverflow-FE7A16?style=for-the-badge&logo=stack-overflow&logoColor=white)](https://stackoverflow.com/questions/tagged/modin) | [![Twitter](https://img.shields.io/twitter/follow/modin_project?style=social&logo=twitter)](https://twitter.com/modin_project) |
```

Comment:
Do some of the buttons appear oversized for you compared to the Twitter one?
![Screen Shot 2022-06-08 at 8 01 47 PM](https://user-images.githubusercontent.com/5554675/172755101-32bac225-2acb-4325-a192-dd41fe07cc49.png)

Reply:
Should we moves docs up here too or keep just the socials?

Reply:
I could imagine the grid being any links that are "actionable" while the badges are more around metrics.

Reply:
I considered that ordering as well. I will add Docs to the table and we can compare.